pages/linux/slop.md

```diff
@@ -0,0 +1,24 @@
+# slop
+
+> Get a selection of the screen.
+> More information: <https://github.com/naelstrof/slop>.
+
+- Wait for the user to make a selection and output its geometry to standard output:
```

Comment:
Perhaps this will read better?
```suggestion
- Wait for the user to make a selection and output its geometry to the standard output:
```

Reply:
I haven't seen it commonly written as "the standard output" in my experience. It's always been either "standard output" or "stdout."